spec/features/documentation_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.feature "Documentation" do
+  scenario "GDS staff visits the docs page" do
+    when_i_visit_the_documentation_page
+    then_i_see_the_documentation_page
```

Comment:
Is the plan to add a spec that loops through the formats defined in the yaml and asserts that they are mentioned here in the documentation?

Reply:
Not really - I think if the page becomes more complicated there will be more tests, but this seems okay-ish for now.